libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+    UpdateStatementAst,
+    WhereClauseAst,
+    WindowClauseAst,
+    WindowClauseContentAst
+} from "./postgresAst";
+
+const LineComment = createToken({name: 'LineComment', pattern: /--.*/, group: 'comments'})
+const BlockComment = createToken({name: 'BlockComment', pattern: /\/\*[^]*?\*\//, line_breaks: true, group: 'comments'})
+const WhiteSpace = createToken({name: 'WhiteSpace', pattern: /\s+/, group: Lexer.SKIPPED})
+
+const Identifier = createToken({name: 'Identifier', pattern: /\b[a-zA-Z_]\w*\b|"([^\\"]|\\\\|\\")+"/})
+const String = createToken({name: 'String', pattern: /E?'([^']|'')*'/i})
```

Biome

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)